test/json_files/error.json

```diff
@@ -0,0 +1,35 @@
+{
+    "_context_roles": [
+        "pony",
+        "unicorn"
+    ],
+    "_context_request_id": "req-magic",
+    "event_type": "not.enough.monads.error",
+    "_context_service_catalog": [
+        {
+            "endpoints": [ ],
+            "endpoints_links": [ ],
+            "type": "monad",
+            "name": "curry"
+        }
+    ],
+    "timestamp": "1970-01-01 00:00:00.000000",
+    "_context_user": "haskell",
+    "_unique_id": "mu",
+    "_context_instance_lock_checked": false,
+    "_context_user_id": "aleph",
+    "payload": {
+        "state_description": "functoring",
+        "code": 999,
+        "message": "ran out of monad while trying to functor the categories",
+        "display_name": "⚓",
```

Comment:
This is a good idea, we should start including non-ASCII characters in more of our test data.